src/components/marquee-wizard/marquee-wizard.component.ts

```diff
@@ -89,6 +89,23 @@ export class MarqueeWizardComponent extends WizardComponent implements OnDestroy
         }
     }
 
+
+    /**
+     * If the validation is true then disable the next button
+     * and do not allow progression onto the next step
+     */
+    isDisabled(): boolean {
+
+         const step = this.getCurrentStep() as MarqueeWizardStepComponent;
```

Comment:
This could probably be simplified to:

```typescript
return !this.getCurrentStep().valid;
```

It might also be good to rename this function something slightly more descriptive. `isDisabled` doesn't give me any indication that this is to do with the next button for example